stable/aws-node-termination-handler/templates/_helpers.tpl

```diff
@@ -24,6 +24,20 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Common labels
+*/}}
+{{- define "aws-node-termination-handler.labels" -}}
+app.kubernetes.io/name: {{ include "aws-node-termination-handler.name" . }}
+helm.sh/chart: {{ include "aws-node-termination-handler.chart" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+k8s-app: aws-node
```

Comment:
I think you copied this from the `aws-vpc-cni`😅

Reply:
LOL, yep. Will fix.